corehq/apps/app_manager/suite_xml/sections/remote_requests.py

```diff
@@ -90,12 +90,19 @@ def get_module_contributions(self, module):
                         RemoteRequestQuery(
                             url=absolute_reverse('remote_search', args=[domain]),
                             storage_instance=RESULTS_INSTANCE,
-                            data=[
+                            data=([
                                 QueryData(
                                     key='case_type',
                                     ref="'{}'".format(module.case_type)
                                 ),
-                            ],
+                                QueryData(
+                                    key='include_closed',
```

Comment:
one thought i had was whether this should just be `closed` to match the value in `CommCareCase`. I think if you made a case property called `include_closed` it would conflict with this which is unlikely but would be confusing. After I had that thought I started thinking that maybe we should namespace either the "special" case properties or the non special ones. e.g `cc:include_closed` or `property-foo`. but I think we can decide on that later. cc @proteusvacuum